### DISCUSSION TITLE
Is it possible to use RailsUI along with something like Jumpstart Pro?

### DISCUSSION THREAD

**Discussion by brandonzylstra** [getrailsui/railsui#25] [Feature requests]
Since Rails UI bills itself as

> Professionally designed UI components & templates made exclusively for Ruby on Rails

I initially assumed that I could use it along with a more general starter kit for a Rails app, like Jumpstart Pro.  But the more I looked into it, it seems more like RailsUI is a competing starter kit, but one more focused on themes, and appearance in general.

Is it possible to use the 2 together?  If not at the moment, will this be supported in the future?

How much work would it be for the user of RailsUI and Jumpstart Pro (the developer of a Rails app) to use them together?

**Comment by justalever** [comment]
@brandondrew, great insights. Yes, Rails UI is meant to solve more of the UI/UX problem, and there are plenty of starter kits out there already, so I didn't feel the need to make another. 

It probably could be used successfully alongside Jumpstart or similar starter kits. I still need to vet Rails UI against them all to give a great answer, but it's definitely on my list of things to hash out. The tricky issue is asset handling and underlying tech in use. Rails UI is best to start with a brand new Rails app. I know Jumpstart preaches the same philosophy. So it's a matter of figuring out if things need to be fixed by installing and using both.
